elixir/apps/web/test/web/acceptance/auth/userpass_test.exs

```diff
@@ -230,6 +232,8 @@ defmodule Web.Acceptance.Auth.UserPassTest do
     # And then to a client
     session
     |> password_login_flow(account, identity.provider_identifier, password, redirect_params)
+    |> assert_el(Query.text("Sign in successful"))
```

Comment:
Can these assertions be flaky? They assert on the temporary page content but the redirect can happen before it's visible by the test process. Will acceptance tests continue to work if we keep only old assertions?

Reply:
So, I added a delay in the redirect message (for the `:test` env only), because there were various scenarios where things didn't seem to be happy in the test environment (both in acceptance tests and unit tests) with a redirect of 1ms.  Since adding that delay in the process message being sent I haven't seen any flakiness in the tests to this point.

I'm guessing the acceptance tests would work if we kept only the old assertions, I just figured this gave us a little more confidence that the redirect page is actually doing what we intended it to do.  If there is a strong concern or there is any flakiness spotted, I can remove the extra assertions.

Reply:
Let change default timeout to 100ms and try that, maybe there is a race and 100ms timeout won't affect any users.